toolbar/toolbar-feature-flags-sveltekit/src/routes/+page.server.ts

```diff
@@ -0,0 +1,8 @@
+import { shouldUseSvelteColor } from '$lib/flags'
+
+export async function load() {
+  return {
+    // Requests the feature flag and returns its value
+    useSvelteColor: await shouldUseSvelteColor(),
+  }
+}
```

Comment:
Have you tested this without the `FLAGS_SECRET` env var locally? I expect people will clone this without pulling env vars, and it would be great to show some instructions within the app in case the env var is missing

Reply:
The README has a step by step guide what you should do to clone and run it locally, which includes the generation + pulling of the env vars

Reply:
If it's not even building for dev without the secret we're good <picture data-single-emoji=":thumbs:" title=":thumbs:"><img class="emoji" width="20" height="auto" src="https://single-emoji.vercel.app/api/emoji/eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..VcBXMJFSW5Dhijgx.-EAtlM54B7kmpit6OHoCdamiG9Ec-D9EjuxG_FxvY1gjBO9r2fmYAZ8g_lWsz-LRIZbL-XeX0nIlNS0_V2i_6THEw_T8RyoG7ogg_JfkkujQmfTpsYTxCQ.lECh7zjM1qtN1xZFZH6XhA" alt=":thumbs:" align="absmiddle"></picture>